frontend/packages/console-shared/src/components/editor/YAMLEditorToolbar.scss

```diff
@@ -1,11 +1,13 @@
+@import '~bootstrap-sass/assets/stylesheets/bootstrap/variables';
+
 .ocs-yaml-editor-toolbar {
   &__links {
     display: flex;
     flex-shrink: 0;
     justify-content: flex-end;
     margin: 2px;
     z-index: 1;
-    @media(max-width: 767px) {
+    @media(max-width: $screen-sm-max) {
```

Comment:
The downside to this fix is it also hides the `View shortcuts` link at viewport sizes 768 - 991, which seems like a regression in functionality.  @spadgett, thoughts?

Reply:
This is how the original `yaml-editor` used to behave. I checked it on a 4.4 cluster as well. There was no other logic to toggle `showSidebar` link based on the window width apart from this css - https://github.com/openshift/console/blob/master/frontend/public/components/_edit-yaml.scss#L42

Reply:
Can you double check?

Without your change I see the links because the toolbar is visible **starting at 768px**.
<img width="1687" alt="Screen Shot 2020-04-06 at 2 33 25 PM" src="https://user-images.githubusercontent.com/895728/78592852-b7b49b80-7813-11ea-9113-d2cc6a236d68.png">

With your change I do **not** see the links because the toolbar is visible **starting at 992px**.
<img width="1713" alt="Screen Shot 2020-04-06 at 2 33 02 PM" src="https://user-images.githubusercontent.com/895728/78592976-df0b6880-7813-11ea-9377-39e1e811d04c.png">

Note the regression is specific to viewport sizes 768 - 991.

Reply:
Right, when I ported the CSS changes for the links component I added `768px` by mistake instead of `991px` which was the original value for `$screen-sm-max`. So originally the toolbar was only visible above 991px screen size. That is why I reverted back to the original value in this PR.

Reply:
You can check the code from `release-4.4` branch as well - https://github.com/openshift/console/blob/release-4.4/frontend/public/components/_edit-yaml.scss#L44